src/components/form/examples/row-layout-schema.ts

```diff
@@ -0,0 +1,107 @@
+export const schema = {
+    description: 'This form has the row layout',
+    type: 'object',
+    properties: {
+        info: {
+            title: 'Preferences',
+            description:
+                'These settings will not affect how others see the data. These are only for you.',
+            type: 'object',
+            lime: {
+                layout: {
+                    type: 'row',
+                },
+            },
+            properties: {
+                language: {
+                    type: 'string',
+                    icon: 'globe',
```

Comment:
You cannot add `icon` here since it is not part of the JSON Schema specification. Any modifications needs to be done under the `lime` schema property